cmd/kubeadm/app/apis/kubeadm/v1beta1/bootstraptokenstring.go

```diff
@@ -31,8 +31,8 @@ import (
 // of view and as an authentication method for the node in the bootstrap phase of
 // "kubeadm join". This token is and should be short-lived
 type BootstrapTokenString struct {
-	ID     string
-	Secret string
+	ID     string `json:"id"`
+	Secret string `json:"secret"`
```

Comment:
Technically speaking, these fields should not be marshalled as they are. That's why `BootstrapTokenString` implements the `json.Marshaler` interface. Hence, they should simply be omitted (`json:"-"`).
Another possibility is to add NOP tags - `json:"ID"`, `json:"Secret"`.
Although, IDK what are the rules and constraints of the verifier spec, that you are using.

Reply:
`-` works for me and the verification script I've used. I will update this PR, thanks for the note